package.json

```
@@ -9,7 +9,7 @@
     "lint": "npm run flow && eslint .",
     "commitmsg": "commitlint -e $GIT_PARAMS",
     "flow": "flow",
-    "build": "rm -r lib/ && babel src/ --out-dir lib/ --copy-files --ignore ___tests___",
+    "build": "babel src/ --out-dir lib/ --copy-files --ignore ___tests___",
```

Can I request an alternative? I'd prefer this to still "clean" before building, adding the `-f` flag will ignore if lib doesn't exist.
```suggestion
    "build": "rm -rf lib/ && babel src/ --out-dir lib/ --copy-files --ignore ___tests___",
```

---

Deal 👍